### PR TITLE
fix: JVM support on Kube

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

File: Dockerfile.runner [hunk 1]
```diff
@@ -7,10 +7,7 @@ COPY ./bin /src/bin
 ENV PATH="/src/bin:$PATH"
 ENV HERMIT_STATE_DIR=/hermit
 RUN hermit uninstall jbr
-RUN hermit install openjre-18.0.2.1_1
-# openjre and jbr conflict, but we want the JRE in the cache
-RUN hermit uninstall openjre
-RUN hermit install jbr
+RUN hermit install openjdk-17.0.8_7
```
Comment (deniseli) [inline Dockerfile.runner:10]: Just curious - are there any good ways to templatize the version number in docker files so you don't have to hardcode the full version number 4 times?

**Review by deniseli — APPROVED** [review]

File: Dockerfile.runner (hunk 1 above)
Comment (stuartwdouglas) [reply Dockerfile.runner:10]: I could probably do it with a build arg.

**Review by safeer — APPROVED** [review]
Nice

File: Dockerfile.runner (hunk 1 above)
Comment (safeer) [reply Dockerfile.runner:9]: Can we land as is and follow up on the build arg? I'd love to use this for the smoketest 😁

Comment (deniseli) [reply Dockerfile.runner:9]: @safeer sgtm :D

**Pull request merged**